conf/storm.yaml.example

```diff
@@ -46,3 +46,12 @@
 #     parallelism.hint: 1
 #     argument:
 #       - endpoint: "metrics-collector.mycompany.org"
+
+## Cluster Metrics Consumers
+# storm.cluster.metrics.consumer.register:
+#   - class: "org.apache.storm.metric.LoggingClusterMetricsConsumer"
+#   - class: "org.mycompany.MyMetricsConsumer"
+#     argument:
+#       - endpoint: "metrics-collector.mycompany.org"
```

Comment:
@harshach 
This is just an example which is same as above, `Metrics Consumer`. Class name could be changed to represent Cluster Metrics but it's just an example so I guess that would be fine.